src/inspector/view/widgets/bendgridcanvas.cpp

```diff
@@ -420,7 +420,12 @@ void BendGridCanvas::hoverMoveEvent(QHoverEvent* event)
     auto oldPointIndex = m_hoverPointIndex;
 
     QRectF frameRect = this->frameRect();
-    QPointF coord = this->frameCoord(frameRect, event->pos().x(), event->pos().y());
+#ifdef MU_QT5_COMPAT
+    QPoint pos = event->pos();
+#else
+    QPoint pos = event->position().toPoint();
```

Comment:
In general, I often don't like `.toPoint()`. With the new API, it makes more sense to stay in the `double` world instead of converting back to `int` and then to `double` again. 
Also, converting from QPoint to QPointF can be done implicitly, so you can use the `QPointF` type regardless of whether it's Qt 5 or 6.

Reply:
changed...